projects/plugins/search/readme.txt

```diff
@@ -16,9 +16,9 @@ Incredibly powerful and customizable, Jetpack Search helps your visitors instant
 
 Jetpack Search is an easy-to-use WordPress search plugin that enhances the default site search functionality provided by the CMS. Thanks to advanced site search tools, your visitors will experience the best search experience you can offer.
 
-From a predictive autocomplete algorithm, to instant search results, Jetpack Search has all the features and options for best site search usability and results.
+From real-time search faceting , to instant search results, Jetpack Search has all the features and options for best site search usability and results.
```

Comment:
Maybe "filtering and faceting".

I think "facets" are not a very well known term.